libs/cardano-ledger-binary/CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 # Version history for `cardano-ledger-binary`
 
-## 1.3.4.1
+## 1.4.0.0
 
-*
+### `testlib`
+
+* Re-export types `Doc` and `AnsiStyle` in `Test.Cardano.Ledger.Binary.TreeDiff`
+* Rename functions:
+  - `diffExprNoColor` -> `diffExprString`
+  - `diffExprNoColor` -> `diffExprCompactString`
```

Comment:
I don't think you can rename one function into two different ones :smile: 

It should be.
* Remove ...
* Add ...

Renaming is only applicable when neither the type signature nor the functionality have experienced any change